frappe/desk/page/backups/backups.js

```diff
@@ -16,7 +16,7 @@ frappe.pages['backups'].on_page_load = function (wrapper) {
 		});
 	});
 
-	page.add_inner_button(__("Get Encryption Key"), function () {
+	page.add_inner_button(__("Get Backup Encryption Key"), function () {
 		if (frappe.session.user === "Administrator") {
```

Comment:
Why is Admin user validation needed for this?

System manager role would be enough right?